src/want-manager/msg-queue.js

```diff
@@ -7,13 +7,19 @@ const logger = require('../utils').logger
 const { wantlistSendDebounceMs } = require('../constants')
 
 module.exports = class MsgQueue {
+  /**
+   *
+   * @param {PeerId} selfPeerId
+   * @param {PeerId} otherPeerId
+   * @param {Network} network
+   */
   constructor (selfPeerId, otherPeerId, network) {
     this.peerId = otherPeerId
     this.network = network
     this.refcnt = 1
 
     this._entries = []
-    this._log = logger(selfPeerId, 'msgqueue', otherPeerId.toB58String().slice(0, 8))
+    this._log = logger(selfPeerId, 'msgqueue')
```

Comment:
Not sure what to do with the last argument here, because `logger` seems to take just two. I dropped it to retain the current behavior, but maybe concatenating last two would make more sense.